src/main/java/uk/ac/ebi/eva/commons/models/data/VariantSourceEntry.java

```diff
@@ -82,7 +83,10 @@ public VariantSourceEntry(String fileId, String studyId) {
     public VariantSourceEntry(String fileId, String studyId, String[] secondaryAlternates, String format) {
         this.fileId = fileId;
         this.studyId = studyId;
-        this.secondaryAlternates = secondaryAlternates;
+        this.secondaryAlternates = Arrays.stream(secondaryAlternates)
```

Comment:
We should call setSecondaryAlternates here...

Reply:
done